api-client/src/utils/GeoJson.ts

```diff
@@ -0,0 +1,22 @@
+import { Feature, FeatureCollection } from 'geojson'
+import LocationModel from '../models/LocationModel'
+
+export const convertToPoint = (location: LocationModel): Feature => {
+  return {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [Number(location.longitude), Number(location.latitude)]
+    },
+    properties: {
+      name: location.name
+    }
+  }
+}
+
+export const embedInCollection = (features: Feature[]): FeatureCollection => {
```

Comment:
I think it would be nice to export only a single function here which has the signature: `LocationModel[] => FeatureCollection`.

The `convertToPoint` could actually be defined on the LocationModel class. Then it can be used as:

`locationModel.convertToPoint()`